content/resources/user/data-types.md

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
Could you add this data types page to the `config.yaml` file in the root directory so it shows up in the ToC on the user pages?